### ISSUE TITLE
Development Canto classifies Fusarium graminearum as a host organism

### ISSUE THREAD

**Issue by jseager7** [pombase/canto#1699]
When I tried to create a multi-allele _Fusarium graminearum_ genotype on the page `pathogen_genotype_manage`, I got redirected to `host_genotype_manage`, rather than staying on `pathogen_genotype_manage` as expected. The inverse of this bug doesn't happen on the `host_genotype_manage` page (I don't get redirected to `pathogen_genotype_manage`).

Having looked at the code, it seems Canto thinks _Fusarium graminearum_ is a host organism: the `getGenotypeManagePath` function receives an `organismType` of `'host'` in this case. I've tested with another pathogen _Alternaria alternata_ [5599], and that still has the correct `organismType` of `'pathogen'`.

I hypothesised that this could be something to do with the remapping of subspecies and strains to species level, so I tried to add two UniProt KB IDs from an _Alternaria alternata_ subspecies (_Alternaria alternata_ f. sp. _mali_ [1970213]), confirmed that Canto mapped the organism back to species-level (_Alternaria alternata_), and created a multi-allele genotype. The `organismType` was correctly set as `'pathogen'` in this case.

I think the bug might be unique to the remapping of _Gibberella zeae_ (strain PH-1)  [229533] &rarr; _Gibberella zeae_ [5518]. However, neither taxon ID 229533 nor 5518 are listed as hosts in the configuration file, so Canto should view them as pathogens by default. The remapping configuration for taxon ID 5518 seems correct too:

```yaml
  5518:
    reference_strain: 5518
    other_strains:
      - 229533
      - 449239
      - 1450154
```

My only suspicion is that taxon ID 55188 _is_ listed as a host in the configuration file, so maybe Canto is somehow confusing this with taxon ID 5518? This doesn't seem very likely though, because surely we would see many more mis-classification of hosts if the parsing of the list was wrong.

I'll try to debug the setting of `$scope.organismType` in `getOrganismType(genotypes)` and see if that reveals anything.

**Comment by jseager7** [comment]
I've now tested this on the production copy of Canto, and the bug doesn't actually occur there, so I suspect it's due to a configuration or data issue on my local development environment (see #1696). I'm dropping the priority of this issue.

**Comment by jseager7** [comment]
Now that I've applied the fix in #1696 I can't reproduce this error, so I suspect that bug was the cause of this one. Closing now.
